ai_agents/devable_research_agent/lib/twitter_collector.py

```
@@ -0,0 +1,472 @@
+    return env
+
+
+def _run_twitter_cli(args: List[str], env: Dict[str, str]) -> Tuple[Optional[Any], Optional[str]]:
+    """Run twitter-cli and parse JSON stdout."""
+    binary = shutil.which("twitter")
+    if not binary:
+        return None, "twitter-cli not found on PATH (install twitter-cli / Agent-Reach)"
+
+    cmd = [binary, *args, "--json"]
+    try:
+        proc = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)